src/screens/NewsScreen/Search.js

```diff
@@ -54,7 +54,7 @@ const styles = StyleSheet.create({
   },
   wrapperSearch: {
     flex: 1,
-    backgroundColor: '#F5F5F5',
+    backgroundColor: COLORS.lightgrey,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change also improves maintainability by replacing the hardcoded color value with a reference from the `COLORS` object.
```diff
-    backgroundColor: '#F5F5F5',
+    backgroundColor: COLORS.lightgrey,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->